cdk/lib/__snapshots__/frontend.test.ts.snap

```diff
@@ -261,6 +261,45 @@ Object {
       },
       "Type": "AWS::IAM::Policy",
     },
+    "DefaultPromotionsLoadingFailureAlarm31718167": Object {
```

Comment:
why are we checking in generated code? is it not possible to generate as part of the build?

Reply:
I think this is standard with cdk

Reply:
fair enough I have raised it on devex channel for more opinions